cmd/plugins/main.go

```diff
@@ -85,10 +86,14 @@ func main() {
 	if err != nil {
 		klog.Fatalf("failed to listen: %v", err)
 	}
+	gatewayK8sClient, err := versioned.NewForConfig(config)
+	if err != nil {
+		panic(err.Error())
```

Comment:
we used `klog.Fatalf` earlier, should we keep it same?

Reply:
Good catch updated to fatal logging.